src/MosaicViews.jl

```diff
@@ -272,4 +279,37 @@ function mosaicview(A::AbstractArray{T,N},
                npad=npad, nrow=nrow, ncol=ncol, rowmajor=rowmajor)
 end
 
+function mosaicview(As::AbstractVector{T},
+                    fill=zero(eltype(first(As)));
+                    kwargs...) where {T <: AbstractArray}
+    length(As) == 0 && throw(ArgumentError("The given vector should not be empty"))
+    N = ndims(first(As))
+    2 <= N || throw(ArgumentError("The given array must have dimensionality of N=2 or higher"))
+    mosaicview(_padded_cat(As; fill=fill, dims=N+1), fill; kwargs...)
+end
+
+function mosaicview(As::Tuple{T, Vararg{T}},
```

Comment:
Currently, we can't add support usage like `mosaicview(X1, X2, X3)` because `fill` is a positional argument.

Thus I'd like to switch `fill` to a keyword argument; I heard that not using keyword argument in pre-1.0 ages is because it was slow.

Thus a deprecation is needed.

cc: @Evizero